.github/workflows/velox_backend.yml

```diff
@@ -163,7 +169,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ "centos:7", "centos:8" ]
```

Comment:
hi @zhouyuan 

why centos 7 gets excluded?

Reply:
It's the same issue as gluten vcpkg builder - node 20 requires more recent libc. The fix is to rewrite the job with checkout on Ubuntu 20.04 + docker run w/ Centos 7. I will fix it in the next patch so it wont block other pull requests